gobbler-server.rkt

```diff
@@ -69,7 +69,7 @@ waiting list.
 ;; An Outgoing is a (make-mail iworld? Server2ClientMessage)
 ;; Represents a message that is to be sent to the iworld
 
-(struct turkey [loc food-eaten waypoint] #:transparent)
+(struct turkey [loc food-eaten [waypoint #:mutable]] #:transparent)
```

Comment:
The entire turkey struct should probably be mutable. We're going to need to update its location and number of foods eaten in other functions. Might as well do it all in one fell swoop.